cvat/apps/engine/media_extractors.py

```diff
@@ -72,6 +72,48 @@ def save_image(self, k, dest_path):
         image.close()
         return width, height
 
+class PDFExtractor(MediaExtractor):
+    def __init__(self, source_path, dest_path, image_quality, step=1, start=0, stop=0):
+        if not source_path:
+            raise Exception('No PDF found')
+        from pdf2image import convert_from_path
+        self._temp_directory = tempfile.mkdtemp(prefix='cvat-')
+        super().__init__(
+            source_path=source_path[0],
```

Comment:
Why source_path[0]?

Reply:
I was following the implementation for `VideoExtractor` See: https://github.com/opencv/cvat/blob/5104cc08c13d325e172d8523b7d2b7347c61c87d/cvat/apps/engine/media_extractors.py#L111

Reply:
I'm not sure why the `source_path` is a list here, but I believe due to the implementation, this will only get a single path at a time.

Reply:
@benhoff , you specified in description of pdf extractor that multiple pdf documents can be uploaded. For video extractor unique flag is True.

Reply:
Extractor's constructor always receive a list as `source_path` argument. I don't see any problem here, but the extractor is responsible to correctly handle passed source list. Could you please adjust description according to extractor behaviour? I mean case if you try to create task with several pdf files but only one will be used. 
```
'pdf': {
  ...
  'unique': **False**
},
```

Maybe it will be better to change behaviour and pass to the constructor a list or single item according its description. I'll think about that.

Reply:
I changed `unique` to `True`. For my case, PDF's can have multiple pages and I want to be able to flip through them, like a video. Let me know if you think the implementation should be different.